client/ayon_core/plugins/publish/collect_anatomy_instance_data.py

```diff
@@ -399,6 +399,10 @@ def _fill_folder_data(self, instance, project_entity, anatomy_data):
                 "parent": parent_name,
                 "folder": {
                     "name": folder_name,
+                    "path": folder_path,
+                    "type": folder_entity["folderType"],
+                    "parent": parent_name,
+                    "parents": hierarchy,
```

Comment:
I personally think it's confusing that "parent" includes the project name if there are no parents for the folder but `parents` never includes the project name.

We could keep e.g. `hierarchy` to be what it was (all parents excluding project) and add `parents` to be `{project_name}/{hierarchy}` so that `parent` makes sense again?

Reply:
I do agree that if `parent` can be project name then `parents` are confusing. My thinking was that `hierarchy` should probably contain the folder name too, but maybe you're right.

Ad. `"parent"` I don't remember why the project name is used? I do remember there was a discussion but don't remember why we chose this approach?

Reply:
> My thinking was that hierarchy should probably contain the folder name too, but maybe you're right.

Kind of had the same - but by having `{hierarchy}` separate from the folder name itself you can actually use them separately, plus if they were combined then you'd just have `folder[path]`? :)

Reply:
I agree on having `hierarchy` as "hierarchy within the project" - so without project name and without folder name. `parents` should be basically `dirname(folder[path])` and `parent` direct ancestor of the folder?